evmlab/tools/statetests/templates/statetest.py

```diff
@@ -28,11 +28,10 @@ def __dict__(self):
 
 class StateTestTemplate(object):
 
-    def __init__(self, nonce=None, codegenerators={}, datalength=None, fill_prestate_for_tx_to=True, fill_prestate_for_args=False, _config=None):
+    def __init__(self, nonce=None, codegenerators={}, datalength=100, fill_prestate_for_tx_to=True, fill_prestate_for_args=False, _config=None):
```

Comment:
we should probably keep this at None and change the default (should be configurable in ini)